layouts/partner-form/single.html

```diff
@@ -43,6 +47,27 @@
       }
     }
 
+		// if vale is input or selected on step
+		function successfulClick() {
+			parent[index].setAttribute("style", "display:none;");
```

Comment:
cache parent[index] in variable

Reply:
putting this into a variable causes the code not to function properly